PROMETHEUS.md

```
@@ -1 +1 @@
-Available at <https://www.opencost.io/docs/installation/prometheus>
+Available at <https://www.opencost.io/docs/installation/prometheus>
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)